build-xcframework.sh

```diff
@@ -125,7 +127,7 @@ cp ../../$HEADER_PATH/$HEADER_NAME Headers/$FRAMEWORK_LIBRARY_NAME.h
 mkdir Modules
 touch Modules/module.modulemap
 cat <<EOT >> Modules/module.modulemap
-framework module $FRAMEWORK_LIBRARY_NAME {
+framework module $FRAMEWORK_MODULE_NAME {
```

Comment:
It's the name of the module used in Objective-C or Swift. I want it to be imported with the name "AriesFramework" instead of "aries_askar". See https://github.com/conanoc/aries-askar/blob/swift_pr/wrappers/swift/Askar/Sources/Askar/Crypto.swift#L2
It's the convention in Swift, and this will not affect other languages.

Reply:
I was wrong. Swift does not seem to consult this module name when importing a framework. I'll rollback this.

Reply:
done